src/app/frontend/replicationcontrollerlist/replicationcontrollercard.html

```diff
@@ -131,3 +131,4 @@
     </div>
   </md-card-content>
 </md-card>
+<kd-expand-parent></kd-expand-parent>
```

Comment:
Yes, sure. I just thought we might want to actually reuse something like `clear: both` on a number of occasions where Edge fails to expand the parent, but you are right that a css class will do the same job and still be self-explanatory :) thanks